### ISSUE TITLE
openjdk_regression suite fails for openjdk11-openj9 due to no TEST.ROOT override

### ISSUE THREAD

**Issue by andrew-m-leonard** [AdoptOpenJDK/openjdk-tests#525]
When trying to run the openjdk11 openjdk_regression suite, we are not overriding the TEST.ROOT to avoid:
  13:03:05 Exception while calling user-specified class: requires.VMProps
  13:03:05 java.lang.UnsatisfiedLinkError: sun/hotspot/WhiteBox.registerNatives()V
as we do for jdk10.

**Comment by andrew-m-leonard** [comment]
The openjdk_regression/build.xml getOpenjdk task needs to also copy the openjdk110openj9 TEST.ROOT

**Comment by smlambert** [comment]
Right, I added https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/openjdk_regression/openjdk11-openj9/TEST.ROOT, but additionally would need to update https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/openjdk_regression/build.xml#L73 to be 10+ versions.

**Comment by andrew-m-leonard** [comment]
This PR is ready: https://github.com/AdoptOpenJDK/openjdk-tests/pull/526
